ledger/blockdb.go

```diff
@@ -144,6 +144,17 @@ func blockGetCert(tx *sql.Tx, rnd basics.Round) (blk bookkeeping.Block, cert agr
 	return
 }
 
+func blockReplaceIfExists(tx *sql.Tx, blk bookkeeping.Block, cert agreement.Certificate) error {
```

Comment:
It would be a nice thing if we could add some validation at this point : 
The Proto and certdata should be identical to the old ones. If that's not the case, we want to write some warning since we didn't expect that.